TX.h

```diff
@@ -538,15 +538,18 @@ static inline void processSBUS(uint8_t c)
     if ((frameIndex == 23) && (c == SBUS_TAIL)) {
       uint8_t set;
       for (set = 0; set < 2; set++) {
-        PPM[(set<<3)] = ppmWork.sbus.ch[set].ch0 >> 1;
-        PPM[(set<<3)+1] = ppmWork.sbus.ch[set].ch1 >> 1;
-        PPM[(set<<3)+2] = ppmWork.sbus.ch[set].ch2 >> 1;
-        PPM[(set<<3)+3] = ppmWork.sbus.ch[set].ch3 >> 1;
-        PPM[(set<<3)+4] = ppmWork.sbus.ch[set].ch4 >> 1;
-        PPM[(set<<3)+5] = ppmWork.sbus.ch[set].ch5 >> 1;
-        PPM[(set<<3)+6] = ppmWork.sbus.ch[set].ch6 >> 1;
-        PPM[(set<<3)+7] = ppmWork.sbus.ch[set].ch7 >> 1;
+        PPM[(set<<3)] = (((ppmWork.sbus.ch[set].ch0 >> 1) * 5) >> 2);
+        PPM[(set<<3)+1] = (((ppmWork.sbus.ch[set].ch1 >> 1) * 5) >> 2);
+        PPM[(set<<3)+2] = (((ppmWork.sbus.ch[set].ch2 >> 1) * 5) >> 2);
+        PPM[(set<<3)+3] = (((ppmWork.sbus.ch[set].ch3 >> 1) * 5) >> 2);
+        PPM[(set<<3)+4] = (((ppmWork.sbus.ch[set].ch4 >> 1) * 5) >> 2);
+        PPM[(set<<3)+5] = (((ppmWork.sbus.ch[set].ch5 >> 1) * 5) >> 2);
+        PPM[(set<<3)+6] = (((ppmWork.sbus.ch[set].ch6 >> 1) * 5) >> 2);
+        PPM[(set<<3)+7] = (((ppmWork.sbus.ch[set].ch7 >> 1) * 5) >> 2);
       }
+      for(uint8_t i=0; i<16; i++){
+       PPM[i]=(PPM[i]<=108)?0:(PPM[i]-108);
```

Comment:
If you have closer a look at `servoBits2Us()`, you will see that on the RX side, for PPM values <12  and >1011, the step size is scaled up  by a factor of 16. So unless you compensate for this when calculating the input PPM values on the TX side, you will get a disproportionate increase in sensitivity outside the 1000 ... 2000 range.